plot_pixel_values.py

```diff
@@ -22,41 +24,103 @@ def get_opt():
     parser.add_argument('--fit-scaling',
                         action='store_true',
                         help='Fit the dark and scale factor (default=False)')
-
+    parser.add_argument('--plot-fits',
+                        action='store_true')
     parser.add_argument('--n-brightest',
-                        default=128,
+                        default=64,
                         type=int,
-                        help='Plot the N brightest (either 128 or n**2)')
+                        help='Plot the N brightest (either 64 or n**2)')
 
     args = parser.parse_args()
     return args
 
 opt = get_opt()
 
+T_CCD_REF = -19 # Reference temperature for dark current values in degC
+def dark_scale_model(pars, t_ccd):
+    """
+    dark_t_ref : dark current of a pixel at the reference temperature
+    scale : dark current model scale factor
+    returns : dark_t_ref scaled to the observed temperatures t_ccd
+    """
+    scale, dark_t_ref = pars
+    scaled_dark_t_ref = dark_t_ref / np.exp(np.log(scale) / 4.0 * (t_ccd - T_CCD_REF))
```

Comment:
You're right terms `t_ccd` and `T_CCD_REF` in the equation were reversed.  To stay consistent with the use of a scaling as a multiplicative term, use:
```
scaled_dark_t_ref = dark_t_ref * np.exp(np.log(scale) / 4.0 * (T_CCD_REF - t_ccd))
```
The `dark_temp_scale` docs say:
```
    Return the multiplicative scale factor to convert a CCD dark map from
    the actual temperature ``t_ccd`` to the reference temperature ``t_ccd_ref``.
```
In our case we are converting from `T_CCD_REF` to `t_ccd`.  The mica docs should be updated to be more generic like:
```
    Return the multiplicative scale factor to convert a CCD dark map from
    the temperature ``t_ccd`` to another temperature ``t_ccd_ref``.
```